documentation/docs/verifiable_credentials/revoke.mdx

```diff
@@ -11,12 +11,29 @@ keywords:
 import revoke_vc_rs from  '!!raw-loader!./../../../examples/account/revoke_vc.rs';
 import CodeSnippet from '../../src/components/CodeSnippetComponent'
 
-TODO: Explain how Verifiable Credentials are revoked using `credentialStatus` and e.g. `RevocationList2020`.
+The [example](#example) below demonstrates how an issuer can revoke a verifiable credential using the Iota Framework by removing its verification method. But another aspect of it, is how an issuer can easily allow verifiers to check if the credential has been revoked.
+In order to allow this, an issuer can link to a location where a verifier can check the credential status. This can be specified in the [credential status](https://www.w3.org/TR/vc-data-model/#status) property.
 
-## Example
+### Revocation List 2020 {#revocation-list-2020}
```

Comment:
Awesome that you added this, but we should (for now) mention that the framework does not support this yet.